src/test/java/Homework22.java

```diff
@@ -0,0 +1,100 @@
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.LoginPage;
+
+
+public class Homework22 extends BasePage
+{
+    public Homework22(WebDriver givenDriver)
+    {
+        super(givenDriver);
+    }
+//    @Test (dataProvider = "CorrectLoginProvider", dataProviderClass = BaseTest.class)
```

Comment:
in the test script you need extends `BaseTest` not is `BasePage`. In the BaseTest all methods to initialize driver and open chrome browser.  And BasePage is the superclass of element subclasses. 
//HINT: 
public class Homework22 extends BaseTest
{
    @Test (dataProvider = "CorrectLoginProvider", dataProviderClass = BaseTest.class)
    public void renamePlaylist(String email, String password)
    {

        LoginPage loginPage = new LoginPage(driver);
        loginPage.login();

        doubleClickPlaylist();
        enterNewPlaylistName();

        Assert.assertTrue(doesPlaylistNameExist());
    }
}